packages/storybook/stories/INTERNAL/IncomingCallNotification/InboundCalling.stories.mdx

```diff
@@ -0,0 +1,34 @@
+import { Meta } from '@storybook/addon-docs';
+
+<Meta
+  id="inboundcalling"
+  title="Concepts/InboundCalling"
+  parameters={{ previewTabs: { canvas: { disable: true, hidden: true } } }}
+/>
+
+# Inbound Calling
+
+Azure Communication Services UI Library is adding support for Inbound Calling. With this new feature, users will
+be able to see and monitor new incoming calls when signed in to ACS or Teams as a Communication as a Teams User.
+Users will be able to accept with audio, accept with video, and reject incoming calls.
+
+Enabling this feature will allow them to see multiple incoming calls allowing the user to service multiple people
+at once best supporting the people that want to get in touch. In any scenario when your user’s assist multiple people simultaneously and better support those trying to reach them. In any scenario, when your users are logged in, you are able to accept or reject based on your users' needs.
```

Comment:
"Enabling this feature will allow them to see multiple incoming calls allowing the user to service multiple people at once best supporting the people that want to get in touch. "

Just feel a bit hard to split this sentence when reading, maybe adding some comma or "and"

Reply:
"In any scenario when your user’s assist multiple people simultaneously and better support those trying to reach them. In any scenario, when your users are logged in, you are able to accept or reject based on your users' needs."

This one looks a bit repetitive to previous sentences